.buildkite/pipeline.yml

```diff
@@ -13,7 +13,7 @@ steps:
   #################
   - label: "🧪 Build and Test"
     key: "test"
-    command: .buildkite/build-and-test.sh
+    command: "build_and_test_pod"
```

Comment:
Is there a style reason for using the quotes?

I [verified](https://github.com/wordpress-mobile/WordPressKit-iOS/compare/dry/buildkite-actions...mokagio/434-test) that the commands run fine without quotes, too.

Reply:
> a linter for these

🤩 thank you for giving me even more ideas for low end-user value-generating things that I know will want to work on but won't do because there's more important stuff to do first! 🤣